ml/et_keras_application_config.yaml

```diff
@@ -15,4 +15,4 @@ preprocessing:
 branch_prefix: et_
 
 # Output directory
-output_directory: /storage/c/wunsch/Artus_2017-12-02/et_keras_4
+output_directory: /storage/c/wunsch/Artus_2018-02-25/et_keras_1
```

Comment:
gehört das zum PR?

Reply:
nicht wirklich, habe nur den output folder angepasst, das ist auch schon in einem eurer commits